### DISCUSSION TITLE
x-transition seems non-responsive in the latest version of Alpine.js (3.2.3)

### DISCUSSION THREAD

**Discussion by hchiam** [alpinejs/alpine#1949] [7. Bugs (archived)]
### Discussed in https://github.com/alpinejs/alpine/discussions/1945

<div type='discussions-op-text'>

<sup>Originally posted by **hchiam** August 12, 2021</sup>
😢 3.2.3: https://codepen.io/hchiam/pen/XWRGgBd seems immediate.

😃 3.2.2: https://codepen.io/hchiam/pen/wvdOeXR transitions nicely by default.</div>

**Comment by dmwyatt** [comment]
It seems strange that it took over a week after the release of 3.2.3 before someone reported this!

Anyway, I've got multiple projects where just switching the cdn from 3.2.2 to 3.2.3 breaks transitions and going back fixes them.

**Comment by KevinBatdorf** [comment]
@dmwyatt some more context here:

https://github.com/alpinejs/alpine/discussions/1914
